packages/riverpod_lint_flutter_test/test/goldens/lints/avoid_use_ref_inside_dispose.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/widgets.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+final provider = Provider((ref) => 0);
+
+class MyWidget extends ConsumerStatefulWidget {
+  const MyWidget({super.key});
+
+  @override
+  ConsumerState<ConsumerStatefulWidget> createState() => _MyWidgetState();
+}
+
+class _MyWidgetState extends ConsumerState<MyWidget> {
+  @override
+  void dispose() {
+    // ignore: avoid_use_ref_inside_dispose
+    ref.read(provider);
+    // ignore: avoid_use_ref_inside_dispose
```

Comment:
Don't use //ignore
Use //expect_lint

This will fail if the warning is missing, which ignore doesn't do